cookbook/security/form_login_setup.rst

```diff
@@ -472,14 +449,14 @@ for the login page:
             ),
         ),
 
-3. Be Sure /login_check Is Behind a Firewall
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+3. Be Sure check_path Is Behind a Firewall
```

Comment:
Ough, this will be hard to learn to new users: `/login` has to be not secured, but behind a firewall... However, I think we cannot do anything about it.